commands/miscellaneous/getStudentProfile.js

```diff
@@ -6,61 +6,60 @@ const ALL_GUILD_IDS = [BATId, BALId, BAGId, BADId, guildId];
 
 module.exports = {
   data: new SlashCommandBuilder()
-    .setName('getstudentprofile')
-    .setDescription('Gets info of a students across BA servers')
-    .setDefaultPermission(false)
-    .addUserOption(option =>
-      option.setName('user')
-        .setDescription('The user you want to get their profile.')
-        .setRequired(true)),
+      .setName('getstudentprofile')
```

Comment:
oops i think my intellij settings did this. do we have git hooks for the linting and stuff?